### ISSUE TITLE
UnixHttpClientITCase.java:49-51: This should be...

### ISSUE THREAD

**Issue by 0pdd** [amihaiemil/docker-java-api#51]
The puzzle `44-50450105` from #44 has to be resolved:

https://github.com/amihaiemil/docker-java-api/blob/6009d3ca39be5d778b1b0273b635996e3bde46ff/src/test/java/com/amihaiemil/docker/UnixHttpClientITCase.java#L49-L51

The puzzle was created by George Aristy on 15-Mar-18. 

Estimate: 30 minutes, role: DEV.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by amihaiemil** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/amihaiemil/docker-java-api/issues/51#issuecomment-373636881))

@amihaiemil Job [#51](https://github.com/amihaiemil/docker-java-api/issues/51) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#51](https://github.com/amihaiemil/docker-java-api/issues/51) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/771878243). The budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by llorllale** [comment]
@amihaiemil this puzzle clearly has an impediment on #41

**Comment by amihaiemil** [comment]
@0crat out

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
com.amazonaws.http.AmazonHttpClient$RequestExecutor[1113] com.amazonaws.SdkClientException: Unable to execute HTTP request: null
sun.nio.ch.FileChannelImpl[110] java.nio.channels.ClosedChannelException: null
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
com.zerocracy.radars.github.ReSafe[76] java.io.IOException: com.amazonaws.SdkClientException: Unable to execute HTTP request: null
com.amazonaws.http.AmazonHttpClient$RequestExecutor[1113] com.amazonaws.SdkClientException: Unable to execute HTTP request: null
sun.nio.ch.FileChannelImpl[110] java.nio.channels.ClosedChannelException: null
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by amihaiemil** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/amihaiemil/docker-java-api/issues/51#issuecomment-374298214))

@amihaiemil GitHub issue is still open, won't close

**Comment by llorllale** [comment]
@amihaiemil this one is still on my agenda

**Comment by llorllale** [comment]
@0cratt waiting on #41

**Comment by 0crat** [comment]
> @0crat waiting on #41 ([here](https://github.com/amihaiemil/docker-java-api/issues/51#issuecomment-374935646))

@llorllale The impediment for [#51](https://github.com/amihaiemil/docker-java-api/issues/51) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
@0crat waiting on #79

**Comment by 0crat** [comment]
> @0crat waiting on #79  ([here](https://github.com/amihaiemil/docker-java-api/issues/51#issuecomment-381334549))

@llorllale Job [#51](https://github.com/amihaiemil/docker-java-api/issues/51) is already on hold

**Comment by llorllale** [comment]
@0crat waiting on #87

**Comment by 0crat** [comment]
> @0crat waiting on #87 ([here](https://github.com/amihaiemil/docker-java-api/issues/51#issuecomment-383563354))

@llorllale Job [#51](https://github.com/amihaiemil/docker-java-api/issues/51) is already on hold

**Comment by llorllale** [comment]
@0crat waiting on #87

**Comment by 0crat** [comment]
> @0crat waiting on #87 ([here](https://github.com/amihaiemil/docker-java-api/issues/51#issuecomment-385499861))

@llorllale Job [#51](https://github.com/amihaiemil/docker-java-api/issues/51) is already on hold

**Comment by llorllale** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/amihaiemil/docker-java-api/issues/51#issuecomment-392594154))

@llorllale The user @llorllale[/z](https://www.0crat.com/u/llorllale) resigned from [#51](https://github.com/amihaiemil/docker-java-api/issues/51), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#51](https://github.com/amihaiemil/docker-java-api/issues/51) assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/1651520591); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by amihaiemil** [comment]
@0crat waiting

**Comment by 0crat** [comment]
> @0crat waiting ([here](https://github.com/amihaiemil/docker-java-api/issues/51#issuecomment-397961734))

@amihaiemil The impediment for [#51](https://github.com/amihaiemil/docker-java-api/issues/51) was registered successfully by @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
You @charlesmike[/z](https://www.0crat.com/u/charlesmike) need to have one of these roles in order to do "Request order start": `ARC`, `PO`; I'm sorry to say this, but at the moment you've got no roles in this project

**Comment by 0crat** [comment]
@charlesmike[/z](https://www.0crat.com/u/charlesmike) I see that you unassigned this issue; the order is still assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) though; to cancel the order use `refuse`, as in [§6](http://www.zerocracy.com/policy.html#6)

**Comment by charlesmike** [comment]
> @charlesmike[/z](https://www.0crat.com/u/charlesmike) I see that you unassigned this issue; the order is still assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) though; to cancel the order use `refuse`, as in [§6](http://www.zerocracy.com/policy.html#6)



@0crat sorry I do not understand you. See [this](http://www.amihaiemil.com/2017/05/23/meet-charles-michael.html) blog post for more details.

**Comment by 0crat** [comment]
Are you speaking to me or about me [here](https://github.com/amihaiemil/docker-java-api/issues/51#issuecomment-427573053); you must always start your message with my name if you want to address it to me, see [§1](http://www.zerocracy.com/policy.html#1)

**Comment by amihaiemil** [comment]
@0crat refuse as I am working on other projects

**Comment by 0crat** [comment]
> @0crat refuse as I am working on other projects ([here](https://github.com/amihaiemil/docker-java-api/issues/51#issuecomment-584039699))

@amihaiemil The user @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) resigned from [#51](https://github.com/amihaiemil/docker-java-api/issues/51), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged (job refused in 14538 hours), see [§6](http://www.zerocracy.com/policy.html#6): -20 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) everybody who has role `DEV` is banned at [#51](https://github.com/amihaiemil/docker-java-api/issues/51); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) everybody who has role `DEV` is banned at [#51](https://github.com/amihaiemil/docker-java-api/issues/51); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
The job [#51](https://github.com/amihaiemil/docker-java-api/issues/51) assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/0a8a4184-67bd-4fc7-b10b-c620afec50c2); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
The user @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) resigned from [#51](https://github.com/amihaiemil/docker-java-api/issues/51), please stop working. Reason for job resignation: It is older than 10 days, see [§8](http://www.zerocracy.com/policy.html#8)
